veps/sig-compute/10-dra-devices/vep.md

```diff
@@ -354,24 +433,33 @@ Note: The feature gate verification when VMI requesting DRA device will either b
 
 1. If devices are requested using DRA, virt controller needs to render the virt-launcher manifest such that
    `pod.spec.resourceClaims` and `pod.spec.containers.resources.claim` sections are filled out.
-1. virt-controller needs a mechanism to watch for virt-launcher pods, resourceclaims and resourceslices to populate the
+   - For GPUs and HostDevices: claims from `vmi.spec.resourceClaims[]` referenced by devices
+   - For Network devices: claims from `vmi.spec.resourceClaims[]` referenced by `vmi.spec.networks[].resourceClaim` 
+2. virt-controller needs a mechanism to watch for virt-launcher pods, resourceclaims and resourceslices to populate the
    `vmi.status.deviceStatus` using the steps mentioned in above section that has all the attributes (for example the
    pciAddress for the gpu device):
     1. The pod status has information about the allocated/reserved resourceClaim.
-    1. The resourceClaim has information about the individual requests in the claim and their allocated device names.
-    1. The resourceslice corresponding to the node running the VMI has information about the allocated device.
+    2. The resourceClaim has information about the individual requests in the claim and their allocated device names.
+    3. The resourceslice corresponding to the node running the VMI has information about the allocated device.
 
 ### Virt launcher changes
 
 1. For devices generated using DRA, virt-launcher needs to use the vmi.status.deviceStatus to generate the domain xml
    instead of environment variables as in the case of device-plugins
-1. The standard env variables `PCI_RESOURCE_<deviceName>` and `MDEV_PCI_RESOURCE_<deviceName>` may continue to be set
+2. The standard env variables `PCI_RESOURCE_<deviceName>` and `MDEV_PCI_RESOURCE_<deviceName>` may continue to be set
```

Comment:
I think in case of DRA we should stop using this one and start adding CDI information